Makefile

```diff
@@ -29,12 +29,22 @@ clean-docs:
 	rm -rf apps/bondy_broker_bridge/doc/*
 	rm -f apps/bondy_broker_bridge/doc/.build
 
+CT_SUITE_FILE?=
```

Comment:
I added this variable to be able to run a single test suite file.
When you type `make test` you can use the bash completion  to get the variable (`CT_[TAB]` will give you 2 choices: `CT_SUITE_FILE` and `CT_SUITE_ARGS`)

I know I can just type `rebar3 ct --suite FILE` but that's kind of a reminder.